month-view.html

```diff
@@ -6,80 +6,101 @@
     <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/smoothness/jquery-ui.css">
     <link rel="stylesheet" href="calendar.css"/>
 </head>
-<body>
+<body onload = "displayMonth()">
 
     <script src="https://code.jquery.com/jquery-3.2.1.min.js" integrity="sha256-hwg4gsxgFZhOsEEamdOYGBf13FyQuiTwlAQgxVSNgt4=" crossorigin="anonymous"></script>
    <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.min.js"
   integrity="sha256-VazP97ZCwtekAsvgPBSUwPFKdrwD3unUfSGVYrahUqU="
   crossorigin="anonymous"></script>
     <script src="./Date.js"></script>
+    <script src="./calendar.js"></script>
```

Comment:
I need you to delete this file and write the html from scratch

Reply:
Do you mean that I should write javascript code in another file for this html page?